iOS/Objective C/LevelPlayDemo/DemoViewController.m

```diff
@@ -161,8 +154,8 @@ - (void) createBannerAd {
 
     // Create the banner view and set the ad unit id and ad size
     if (self.bannerSize != nil) {
-        self.bannerAd = [[LPMBannerAdView alloc] initWithAdUnitId:kBannerAdUnitId];
-        [self.bannerAd setAdSize:self.bannerSize];
+      LPMBannerAdViewConfig *config = [[[[LPMBannerAdViewConfigBuilder alloc] init] setWithAdSize:self.bannerSize] build];
+      self.bannerAd = [[LPMBannerAdView alloc] initWithAdUnitId:kBannerAdUnitId config: config];
 
```

Comment:
"loadAdWithViewController for banner" most comments have : at the end but some don't. Better to align all with : or without